openerp/tools/mail.py

```diff
@@ -123,6 +130,30 @@ def html_sanitize(src, silent=True, strict=False, strip_style=False, strip_class
     return cleaned
 
 
+_style_whitelist = ['width', 'height', 'font-size']
+_style_replace = {
+    ''
+}
+
+def sanitize_attributes(html_root):
```

Comment:
I looked at the source code and I did not found any clean way to plug into the code. There is an allow_element for tags. However concerning attributes, I did not found a method to override. We could append the code to __call__, maybe.